PROTOCOL.md

```diff
@@ -351,6 +351,25 @@ animation) and `|[silent]` (suppress message).
 > turn. Otherwise, it means that the weather has changed due to a move or ability,
 > or has expired, in which case `WEATHER` will be `none`.
 
+`|-fieldstart|CONDITION`
+
+> Field condition `CONDITION` started. This includes all moves that effect the entire
+> field but isn't a weather such as Trick Room or any Terrain move.
+
+`|-fieldend|CONDITION`
+
+> Indicates that the condition `CONDITION` ended.
+
+`|-sidestart|SIDE|CONDITION`
+
+> A side condition `CONDITION`, such as Entry Hazards, Reflect/Light Screen/Safeguard,
+> or Tailwind, is started on `SIDE`. 
+
+`|-sideend|SIDE|CONDITION`
+
+> Indicated when a side condition ends on `SIDE`. `CONDITION` specify which
+> condition that ended.
```

Comment:
I'd like this description to be changed to somewhat resemble with the `-fieldend` description. Something like "... ended for the given `SIDE`" might work nicely.